pkg/controller/provider/container/ovirt/collector.go

```diff
@@ -69,10 +70,18 @@ func New(db libmodel.DB, provider *api.Provider, secret *core.Secret) (r *Collec
 		libpath.Join(
 			provider.GetNamespace(),
 			provider.GetName()))
+	clientLog := logging.WithName("client|ovirt").WithValues(
+		"provider",
+		libpath.Join(
+			provider.GetNamespace(),
+			provider.GetName()))
 	r = &Collector{
 		client: &Client{
-			url:    provider.Spec.URL,
-			secret: secret,
+			url:           provider.Spec.URL,
+			secret:        secret,
+			log:           clientLog,
+			clientTimeout: time.Minute * 30, // TODO: hardcoded for now, move to provider spec
```

Comment:
Please fix this now rather than leaving a `TODO`. The Provider spec contains a `settings` map for provider-type-specific fields, which could be used for this, with a default of 30 minutes if a value is not set.